src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -776,66 +776,5 @@ internal BoundStatement WrapWithVariablesAndLocalFunctionsIfAny(CSharpSyntaxNode
                                   ImmutableArray.Create(statement))
             { WasCompilerGenerated = true };
         }
-
-        internal string Dump()
```

Comment:
I was being a bit ruthless. A lot of the code which appears designed to be run in the debugger don't have #if DEBUG guards, perhaps they should?